.github/workflows/sbom.yaml

```diff
@@ -16,7 +16,7 @@ jobs:
       contents: write
 
     env:
-      OUTPUT: kube-state-metrics-${{ github.event.release.tag_name }}.spdx
+      OUTPUT: sbom.spdx
```

Comment:
CLOMonitor expects asset name to contain `sbom`

https://clomonitor.io/docs/topics/checks/#software-bill-of-materials-sbom